spring-cloud-gcp-samples/spring-cloud-gcp-integration-pubsub-json-sample/src/test/java/com/example/PubSubJsonPayloadSampleApplicationIntegrationTests.java

```diff
@@ -34,17 +43,28 @@
 import org.springframework.core.ParameterizedTypeReference;
 import org.springframework.http.HttpMethod;
 import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.TestPropertySource;
 import org.springframework.test.context.junit.jupiter.SpringExtension;
 
 /** Tests the Pub/Sub Json payload app. */
 // Please use "-Dit.pubsub-integration=true" to enable the tests
-@EnabledIfSystemProperty(named = "it.pubsub-integration", matches = "true")
+// @EnabledIfSystemProperty(named = "it.pubsub-integration", matches = "true")
```

Comment:
Is this intentional disabling of the tests?

Reply:
Thank you! Reverted this change.